packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -7,7 +7,9 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_plugin_android_lifecycle: ^1.0.0
-  google_maps_flutter_platform_interface: ^1.0.1
+  
+  google_maps_flutter_platform_interface: 
+    path: ../google_maps_flutter_platform_interface
```

Comment:
This will make the plugin unpublishable, please rebase with flutter/plugins:master again once the actual google_maps refactor lands!

Reply:
Thanks for the super quick refactor though!

Reply:
Will do. Just needed this to be able to test whether it is working or not.

Reply:
@bnxm makes sense. Note that now that you're touching 2 separate packages, this will eventually have to split into 2 PRs: one for the platform interface changes, and another one for the core plugin.

I'm merging the google_maps_flutter refactor today. It won't be published to pub, but it'll let you do the proper rebase.

Reply:
The refactor has [landed](https://github.com/flutter/plugins/commit/1a9d51d344f71b0f8af657053b04783e274fd1ca).

Reply:
I think I did all the necessary refactorings. Couldn't test it though, because the `platform_interface` is not implemented yet I guess.

Reply:
I don't understand. The platform_interface is already in [pub](https://pub.dev/packages/google_maps_flutter_platform_interface)

Reply:
Sorry, I meant that the `GoogleMapsFlutterPlatform` class in the `platform_interface` is not implemented yet and thus when trying to run on a real device, the map isn't working its current state.